scripts/update_aports.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+set -e
+
+alpine_version=$(cat /etc/alpine-release | cut -d. -f1-2)
+
+ros_distro=${ROS_DISTRO:-noetic}
+short_hash=$(git log --format=%h -n1 ${ros_distro})
+upstream_commit_msg=$(git log --format=%B ${short_hash}^..${short_hash} \
+  | sed ':a; N; $!b a; s/\\/\\\\/g; s/\t/\\t/g; s/"/\\"/g; s/\n/\\n/g; s/\r/\\r/g;')
+aports_slug='seqsense/aports-ros-experimental'
+
+distro_dir=${ros_distro}
+case ${ros_distro} in
+  "noetic")
+    distro_dir=${ros_distro}.v${alpine_version}
+    ;;
+  *)
+    echo "Unsupported ROS_DISTRO"
+    ;;
+esac
+
+generate_opts=
+case "${ALPINE_VERSION}" in
+  3.17) ;;
+  *)
+    generate_opts="${generate_opts} --split-dev"
+    ;;
+esac
+
+branch="rosdistro1-${distro_dir}-${short_hash}"
+
+if git ls-remote --exit-code \
+  https://github.com/${aports_slug}.git \
+  refs/heads/${branch}; then
+  echo "The change is already pushed"
+  exit 0
+fi
+
+# Setup local rosdistro
+
+rosdistro_build_cache index.yaml ${ros_distro}
+export ROSDISTRO_INDEX_URL="file://$(pwd)/index.yaml"
+rosdep update
+
+# Clone and update aports
+
+tmpdir=$(mktemp -d)
+trap "rm -rf $tmpdir" EXIT
+
+git clone https://github.com/${aports_slug}.git ${tmpdir}
+mkdir -p ${tmpdir}/v${alpine_version}/ros/${ros_distro}
+cd ${tmpdir}/v${alpine_version}/ros/${ros_distro}
+
+git config user.name "at-wat"
+git config user.email "8390204+at-wat@users.noreply.github.com"
```

Comment:
```suggestion
git config user.name "alpine-ros-bot"
git config user.email "214657941+alpine-ros-bot@users.noreply.github.com"
```
Use bot email address instead